CollAction/Views/Home/About.cshtml

```diff
@@ -139,3 +139,21 @@
     </div>
 
 </div>
+
+<script type="text/javascript">
+    var shiftWindowToCorrectForNavBarHeight = function ()
+    {
+        var navBarHeight = document.querySelector('.nav-row').offsetHeight;
```

Comment:
@stewartmatheson We should probably either:
* Give poor marc an alternative solution
* Or merge this and fix later

Since you're the front-end expert, I'll leave that to you.